src/components/cards/profile/Referral.tsx

```diff
@@ -25,54 +28,58 @@ export default function Referral({ referral }: ReferralProps): ReactElement {
 
+  const status = (evaluation: Evaluation) => {
+    if (!evaluation) return t("referrals.challenge.evaluation.status.pending")
+    return t(evaluation?.reward ? "referrals.challenge.evaluation.status.passed" : "referrals.challenge.evaluation.status.failed")
+  }
+
+  const formatDate = (date: Date) => {
+    return date ? DateManager.fromNow(date, locale) : null
+  }
   return (
-    <div className="text-sm text-gray-700 bg-gray-50 md:mb-0">
-      <div className="flex p-7">
-        <div className="">
+    <div className="text-sm text-gray-700 bg-gray-50 md:mb-0 p-7">
+      <div className="w-full">
+        <div className="flex gap-4 ">
           <Avatar size="large" user={referral.user} hideVerificationBadge />
-        </div>
-        <div className="ml-5">
-          <span className="pb-1 text-lg font-medium leading-loose text-gray-900">{referral.user?.displayName}</span>
-          <p>
-            {t("referrals.joined")} {joinedAt}
+          <p className="grid">
+            <span className="pb-1 text-lg font-medium leading-loose text-gray-900">{referral.user?.displayName}</span>
+            <span className="text-gray-500">{t("referrals.joined")} {joinedAt}</span>
           </p>
+        </div>
+        <div className="flex-grow md:ml-19">
           <div className="pt-1">
-            <ul className="pb-1 text-sm font-light leading-loose text-gray-700 list">
-              {referral.challenge && referral.community && (
-                <li>
-                  <span className="ml-5">{t("referrals.challenge-participation")}</span>
-                  <span className="font-bold">{referral.community.name}</span>
-                  <span className="hidden md:inline-block">{participatedAt}</span>
-                </li>
-              )}
-
-              {referral.submission && referral.submission.metadata && referral.submission.metadata.evaluation && (
-                <li>
-                  <span className="ml-5">{t("referrals.submission.evaluation")}</span>
-                  <span className="font-bold">
-                    {referral.submission.metadata.evaluation.points}/{referral.submission.metadata.evaluation.totalPoints}
+            <ul className="pb-1 font-light leading-loose text-gray-700 grid divide-y-2 space-y-4 divide-gray-200">
+              {referral?.submissions?.length ? referral.submissions.map((submission) => (
+                <li key={submission.id} className="grid md:flex justify-between pt-4 gap-2 md:gap-0">
+                  <span className="grid gap-2">
+                    <span>
+                      {status(submission.metadata.evaluation)} <Link href={`/communities/${submission.community.slug}/challenges/${submission.challenge.id}/submissions/${submission.id}`} className="font-bold underline text-base- underline-offset-2">{submission.challenge.name} {t("referrals.submission.challenge")}</Link>
```

Comment:
`{`/communities/${submission.community.slug}/challenges/${submission.challenge.id}/submissions/${submission.id}` can you build on top, having it inline does not make readable